benchmarks/longmemeval_v2_release_package_process.py

```
@@ -0,0 +1,362 @@
+            expected_returncode=returncode,
+            expected_invocations=1,
+        )
+        _require_success(returncode, step=step)
+        outputs = collect()
+        require_receipts(root, receipts, secrets=secrets)
+        expected = receipt_files(root, receipts) | {
+            log_path(root, step),
+            *(Path(binding["path"]) for binding in outputs.values()),
+        }
+        require_inventory(root, expected)
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)